### PR TITLE
Feature/select

### LINKED ISSUE #64: Feature/Select
Select box

### PR DESCRIPTION
**Pull request by Snugug**
Oh the crazy.

Resolves #64

### CODE REVIEW

File: patterns/base/select/select.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<select name="demo-select" id="demo-select" class="base--select">
```
Comment (una) [inline patterns/base/select/select.html]: kind of nitpicky so feel free to ignore, but i'd change the name and id to something less closely resembling the class name (such as just "demo" or "select-demo")

Comment (Snugug) [inline patterns/base/select/select.html]: Can do.

**Comment by Snugug** [comment]
Update made

File: patterns/core/sass/functions/_triangle.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+//
+// Creates an SVG triangle
+// @param {Color} $fill - Fill color
+// @param {Color} $stroke - Stroke color
+// @param {Number} $stroke-width - Width of the stroke
+// @param {Number} $stroke-miter - Miter Limit of stroke
+// @param {Number} $size - The size of the triangle
+// @param {String} - $direction - Direction the arrow is pointing. Can be one of `up`, `down`, `left`, `right`
+// @param {Boolean} - $encode - Whether or not to encode string
+// @return {String} - Updated string
```
Comment (una) [inline patterns/core/sass/functions/_triangle.scss]: @dependancies param info

**Pull request merged**